--- a/conacq/eval/__init__.py
+++ b/conacq/eval/__init__.py
@@ -49,7 +49,6 @@
 # Cross-validation
 from .cross_validation import (
     n_fold_cross_validation,
-    n_fold_cross_validation_conmin,
     n_fold_cross_validation_interactive,
     CrossValidationResult,
     CrossValidationFoldResult
@@ -109,7 +108,6 @@
 
     # Cross-validation
     'n_fold_cross_validation',
-    'n_fold_cross_validation_conmin',
     'n_fold_cross_validation_interactive',
     'CrossValidationResult',
     'CrossValidationFoldResult',
